### PR TITLE
#165413125 add super admin functionality for user CRUD

### PR DESCRIPTION
**Pull request by Niyitangasam**
Description
=======
This PR  adds functionalities of helping super admins to able to manage users.

How has it been tested
=======

This was tested through postman using below endpoints:

-  POST ```/api/v1/users/``` to create a new user
-  GET ``` GET /api/v1/users/ ``` to fetch all users
-  PUT ```api/v1/users/{username}``` to update a user information
-  GET ```api/v1/users/{username}``` to fetch single user
-  PUT ```api/v1/users/{username}/active/{disable}``` to deactivate a user

Checklist:
=======
N/A

Pivotal tracker story ID
=======
[165413125](https://www.pivotaltracker.com/n/projects/2326984/stories/165413125)

### CODE REVIEW

File: controllers/user.controller.js [hunk 1]
```diff
@@ -123,6 +123,60 @@ class UserController {
...
+     * @return {json} Returns json object
+     * @static
+     */
+  static async createUser(req, res) {
+    const newUser = await userHelper.createNewUser(req);
+    if (newUser) {
+      return res.status(201).send({
+        status: 201, message: `New User created and the password is ${newUser.generatedPassword}`
+      });
+    }
```
Comment (AJAkimana) [inline controllers/user.controller.js:140]: Put `else` or return something if `!newUser`

Comment (Niyitangasam) [reply controllers/user.controller.js:140]: Done

**Review by Kabalisa — APPROVED** [review]
great job @Niyitangasam

**Review by AJAkimana — APPROVED** [review]
Good work @Niyitangasam

**Pull request merged**